src/generators/jvm/jvmAttribute.ml

```diff
@@ -167,7 +167,7 @@ let rec write_annotation ch ann =
 			| ValClass i ->
 				write_ui16 ch i
 			| ValAnnotation a ->
-				write_annotation ch ann
+				write_annotation ch a
```

Comment:
Oh, good catch! Do you know how to reproduce this? Because this looks like it should cause a stack overflow in the compiler.

Reply:
No, I just saw this while looking for what to change and was like "wait, that's not right"